### PR TITLE
Index name reference object

### PR DESCRIPTION
**Pull request by PayalKhanna**
original contribution https://github.com/finos/common-domain-model/pull/2309

### CODE REVIEW

File: rosetta-source/src/main/resources/cdm-sample-files/fpml-5-10/incomplete-products/credit-derivatives/expectations.json [hunk 1]
```diff
@@ -22,23 +22,23 @@
...
 {"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/credit-derivatives/cd-ex18-standard-north-american-corp.xml","excludedPaths":0,"externalPaths":42,"outstandingMappings":2,"validationFailures":3,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"CreditDefaultSwap_SingleName","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":1}},
-{"fileName":"cdm-sample-files/fpml-5-10/incomplete-products/credit-derivatives/cd-ex19-cdx-index-option-pred-clearing.xml","excludedPaths":0,"externalPaths":82,"outstandingMappings":11,"validationFailures":3,"qualificationExpectation":{"success":true,"qualifyResults":[{"qualifiedName":"CreditDefaultSwap_Index","qualifiedObjectClass":"cdm.product.template.EconomicTerms"},{"qualifiedName":"CreditDefaultSwaption","qualifiedObjectClass":"cdm.product.template.EconomicTerms"}],"qualifiableObjectCount":2}},
```
Comment (hugohills-regnosys) [inline rosetta-source/src/main/resources/cdm-sample-files/fpml-5-10/incomplete-products/credit-derivatives/expectations.json:25]: Qualification is broken

Comment (PayalKhanna) [reply rosetta-source/src/main/resources/cdm-sample-files/fpml-5-10/incomplete-products/credit-derivatives/expectations.json:25]: @winitall01 can you look into this

**Comment by PayalKhanna** [comment]
@hugohills-regnosys @lolabeis can you review the expectation changes please

**Comment by PayalKhanna** [comment]
please close this PR

**Pull request closed without merging**